apps/beeswax/src/beeswax/models.py

```diff
@@ -471,7 +471,17 @@ class Session(models.Model):
   objects = SessionManager()
 
   def get_handle(self):
-    secret, guid = HiveServerQueryHandle.get_decoded(secret=self.secret, guid=self.guid)
+    secret = self.secret
+    guid = self.guid
+    if sys.version_info[0] > 2:
+      # only for py3, after bytes saved, bytes wrapped in a string object
+      # a new guid/secret before saved, the eval() will throws exception
+      try:
+        secret = eval(secret)
+        guid = eval(guid)
+      except:
```

Comment:
Nice!

Just could we just add a test on get_handle()?

Then will be easy to quick check if we can do something slightly cleaner than eval()

For the record on previous commit that broke it: 
https://github.com/cloudera/hue/commit/7d41081166d18a9362a13e332082e5ca64ba7d3a#diff-7eda3aa27bde30ed2e353eca90d753d39a764a4e218676124fd091ead7a97883

Reply:
Added unit test for get_handle()